tests/unit_tests/blas/include/reference_blas_templates.hpp

```diff
@@ -48,13 +49,10 @@ static void (*zrotg_p)(void *a, void *b, const double *c, void *s);
 
 static LIB_TYPE cblas_library() {
     if (h == NULL) {
-#ifdef _WIN64
-        h = GET_LIB_HANDLE("libblas.dll");
-        if (h == NULL)
-            h = GET_LIB_HANDLE("blas.dll");
-#else
-        h = GET_LIB_HANDLE("libblas.so");
-#endif
+        h = GET_LIB_HANDLE(REF_BLAS_LIBNAME);
+        if (h == NULL) {
+            throw std::runtime_error("CBLAS library not found.");
```

Comment:
@mmeterel can you please confirm if `libblas` is a C or Fortran library? I think it is Fortran and the error message with "BLAS" instead of "CBLAS" would be more relevant.

Reply:
I had that confusion too - libblas is a Fortran library. But the function name calling this runtime_error is also called `cblas_library()`. Also, the cmake config file to find `libblas` or `libcblas` is called FindCBLAS.cmake. So, I don't see that we are  very consistent in naming.

Reply:
In that case, I am okay with the error message saying "CBLAS". Thank you.

Reply:
I had the same confusion. Let me know if I should fix this to make it clearer